mmdet3d/models/dense_heads/anchor_free_mono3d_head.py

```diff
@@ -266,29 +273,8 @@ def _init_predictor(self):
             self.conv_attr = nn.Conv2d(self.attr_branch[-1], self.num_attrs, 1)
 
     def init_weights(self):
-        """Initialize weights of the head."""
-        for modules in [self.cls_convs, self.reg_convs, self.conv_cls_prev]:
-            for m in modules:
-                if isinstance(m.conv, nn.Conv2d):
-                    normal_init(m.conv, std=0.01)
-        for conv_reg_prev in self.conv_reg_prevs:
-            if conv_reg_prev is None:
-                continue
-            for m in conv_reg_prev:
-                if isinstance(m.conv, nn.Conv2d):
-                    normal_init(m.conv, std=0.01)
-        if self.use_direction_classifier:
-            for m in self.conv_dir_cls_prev:
-                if isinstance(m.conv, nn.Conv2d):
-                    normal_init(m.conv, std=0.01)
-        if self.pred_attrs:
-            for m in self.conv_attr_prev:
-                if isinstance(m.conv, nn.Conv2d):
-                    normal_init(m.conv, std=0.01)
+        super().init_weights()
         bias_cls = bias_init_with_prob(0.01)
```

Comment:
We can also modify the bias through init_cfg

Reply:
by override

Reply:
We are not sure the self.use_direction_classifier and self.pred_attrs are always True, So we can not use the override.